src/table/src/requests.rs

```diff
@@ -128,6 +129,25 @@ impl TableOptions {
     }
 }
 
+impl fmt::Display for TableOptions {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut key_vals = vec![];
+        if let Some(size) = self.write_buffer_size {
+            key_vals.push(format!("{}={}", WRITE_BUFFER_SIZE_KEY, size));
```

Comment:
Should we quote key values by `""`?
e.g. `"write_buffer_size"="128.0MiB" "ttl"="16m 40s"`

Reply:
Looks like It's better to convert it into `OptionMap`, call its function `kv_pairs`, and join the result. This function will obscure information such as access keys.

https://github.com/GreptimeTeam/greptimedb/blob/be29e48a60345599373128c8cd6632f4f407165c/src/sql/src/statements/option_map.rs#L79

Reply:
It seems mysql/mariadb is writing it without quote

```
MariaDB [(none)]> select create_options from information_schema.tables limit 5;
+--------------------------------------------+
| create_options                             |
+--------------------------------------------+
| max_rows=10380835156842741 transactional=0 |
| max_rows=10720                             |
| max_rows=34952                             |
| max_rows=8587869680497929 transactional=0  |
| max_rows=51150                             |
+--------------------------------------------+
```

Reply:
I don't think we need to keep it compatible with MySQL or Mariadb in this column.